pkgs/by-name/to/toml11/package.nix

```
@@ -1,25 +1,37 @@
-{ lib
-, stdenv
-, fetchFromGitHub
-, cmake
+{
+  lib,
+  cmake,
+  fetchFromGitHub,
+  gitUpdater,
+  ninja,
+  stdenv,
 }:
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "toml11";
-  version = "3.7.1";
+  version = "4.1.0";
```

```suggestion
  version = "4.2.0";
```

https://github.com/ToruNiina/toml11/releases/tag/v4.2.0

Please add this link to the body of commit message.

---

Changelog is also missing.

---

Since some recent decision, parametric changelogs are being avoided, since they are fragile and can break if. e.g. the version is the form `X-unstable-...`.